src/errors.js

```diff
@@ -1,4 +1,5 @@
 const ERRORS = {
+  INVALID_ENCODING_FORMAT: 'Supporting encoding format: hex and base64',
```

Comment:
Supported encoding formats: hex and base64
OR Supporting encoding formats hex and base64